tests/scripts/test_scripts.py

```diff
@@ -723,6 +723,29 @@ def test_train_rl_double_normalization(tmpdir: str, rng):
         )
 
 
+def test_train_rl_cnn_policy(tmpdir: str, rng):
+    venv = util.make_vec_env(
+        "AsteroidsNoFrameskip-v4",
+        n_envs=1,
+        parallel=False,
+        rng=rng,
+    )
+    print(venv)
+    net = reward_nets.CnnRewardNet(venv.observation_space, venv.action_space)
+    tmppath = os.path.join(tmpdir, "reward.pt")
+    th.save(net, tmppath)
+
+    log_dir_data = os.path.join(tmpdir, "train_rl")
+    with pytest.warns(RuntimeWarning):
+        train_rl.train_rl_ex.run(
+            named_configs=["train.cnn_policy"] + ALGO_FAST_CONFIGS["rl"],
+            config_updates=dict(
+                common=dict(log_dir=log_dir_data, env_name="AsteroidsNoFrameskip-v4"),
+                reward_path=tmppath,
+            ),
+        )
+
```

Comment:
Consider checking:
```
    assert run.status == "COMPLETED"
    assert isinstance(run.result, dict)
```

like in `test_train_rl_main`.

Right now I think this test could pass even if there were some exceptions as Sacred will just capture them (but return `run.status == "ERROR"` or the like.)

Reply:
Done.